index.md

```diff
@@ -19,22 +21,13 @@ This site serves as a collection of all the documentation for RAPIDS. Whether yo
 ## Stay Connected
 
 <div class="footer-help-section">
-    <div class="footer-help-box">
-        <div class="footer-help-box-image"><i class="fas fa-users fa-4x"></i></div>
-        <a href="https://groups.google.com/forum/#!forum/rapidsai" target="_blank" class="btn">Google Groups</a>
-    </div>
-    <div class="footer-help-box">
-        <div class="footer-help-box-image"><i class="fab fa-twitter fa-4x"></i></div>
-        <a href="https://twitter.com/rapidsai" target="_blank" class="btn">Twitter</a>
-    </div>
-    <div class="footer-help-box">
-        <div class="footer-help-box-image"><i class="fab fa-slack fa-4x"></i></div>
-        <a href="https://join.slack.com/t/rapids-goai/shared_invite/enQtMjE0Njg5NDQ1MDQxLTViZWFiYTY5MDA4NWY3OWViODg0YWM1MGQ1NzgzNTQwOWI1YjE3NGFlOTVhYjQzYWQ4YjI4NzljYzhiOGZmMGM" target="_blank" class="btn">Slack</a>
-    </div>
-    <div class="footer-help-box">
-        <div class="footer-help-box-image"><i class="fab fa-stack-overflow fa-4x"></i></div>
-        <a href="https://stackoverflow.com/tags/rapids" target="_blank" class="btn">Stack Overflow</a>
-    </div>
+    {% for i in site.social %}
+        {% assign social = i[1] %}
+        <div class="footer-help-box">
+            <div class="footer-help-box-image"><i class="{{ social.fa-icon-class }} fa-4x"></i></div>
```

Comment:
Replace "fa-4x" with "fa-3x"

and edit custom.css line 133+ with:
`/* Help section */
.footer-help-section {
    display: flex;
    justify-content: left;
    padding: 1rem 0;
    flex-wrap: wrap;
}

.footer-help-box {
  padding: 0 1rem;
  text-align: center;
}

.footer-help-box-image img {
  min-height: 82px;
  align-items: center;
  justify-content: center;
  max-height: 82px;
}

.footer-help-box p {
  font-size: 14px;
  display: block;
  padding: 10px 0 0 0;
  font-weight: 300;
  text-transform: uppercase;
  letter-spacing: 0.1em;
}`